### ISSUE TITLE
Add isInStoreAssortmentSearch as query parameter

### ISSUE THREAD

**Issue by alexanderfrodeberg** [AlexGustafsson/systembolaget-api#13]
When performing a search for assortments on systembolaget.se, it seems that the parameter isInStoreAssortmentSearch is used to filter products that are available in the specified store (storeId). Only providing the storeId does not give the expected response, as it seems the isInStoreAssortmentSearch parameter is needed for accurate filtering.

**Comment by fiskhest** [comment]
Found this too last night, specifically when filtering for `Lokalt och Småskaligt` it seems that the store somehow has the product registered, but not as in the store assortment. Maybe because it should be possible to order those beers to the store? 🤔 In any case, I was able to overcome it by modifying `FilterByStore()`:
```
// FilterByStore filters products that are in a specific store's assortment.
func FilterByStore(store string) SearchFilter {
	return func(v *url.Values) {
		v.Set("storeId", store)
		v.Set("isInStoreAssortmentSearch", "true")
	}
}
```

Preparing a PR for maintainer to consider.
